config/session.js

```diff
@@ -9,6 +9,14 @@
  * https://sailsjs.com/config/session
  */
 
+const featureFlags = require('../api/utils/FeatureFlags');
+function generateTls()  {
+  if (featureFlags.isEnabled('FEATURE_REDIS_SSL')) {
+    return { };
+  }
+  return undefined;
+}
+
```

Comment:
Same comment as above about coming back around to this later. Maybe add a @TODO comment block.

Reply:
Same as above, this will get changed in #158